### ISSUE TITLE
setNextMedia weird behavior

### ISSUE THREAD

**Issue by alexcristd** [wang-bin/mdk-sdk#59]
Hello,

Using Qt5 with MinGW 8.1.0 64bit on Windows 10 64bit, and MDK 0.14.2 (nightly as of today).

I attached a sample app.
[seek.zip](https://github.com/wang-bin/mdk-sdk/files/8845670/seek.zip)

I'm experiencing 2 issues with ```setNextMedia```:

1. calling ```setNextMedia()``` twice with a valid string causes the app to freeze. It doesn't crash, but the window freezes, seems like a deadlock or something. In the sample app, if you press Play twice (without pressing Pause between), it will freeze.

A workaround is to call ```setNextMedia(nullptr)``` before calling it with a valid string, and this avoids the freeze. But I think this should be fixed internally.

2. calling ```setNextMedia(nullptr)``` (as per issue (1) ), the video area flickers. In the sample code, pressing pause causes the flicker.

If either of the issues is fixed, the other one becomes less important, but I think both should be fixed to avoid problems in other scenarios.

If you need more details, let me know. Thanks!

**Comment by wang-bin** [comment]
> calling setNextMedia(nullptr) (as per issue (1) ), the video area flickers. In the sample code, pressing pause causes the flicker.

add `SetGlobalOption("videoout.clear_on_stop", 0);`

**Comment by alexcristd** [comment]
Thanks! Seems to fix the flicker.

However, is this the appropriate way it should be done, or is it just a temporary workaround? It puzzles me because playback is not stopped, it's only paused, and the image buffer is not completely cleared, the image remains on the screen while paused, but it just blinks once.

**Comment by wang-bin** [comment]
yes. image won't be cleared if playback is stopped. but if the state is paused, it's a bug
